xcode/commonFastfile

```diff
@@ -220,10 +236,33 @@ lane :ManuallyUpdateCodeSigning do |options|
 
   encryption.encrypt_files
 
-  files_to_commit = Dir[File.join(storage.working_directory, "**", "*.{cer,p12,mobileprovision}")]
+  files_to_commit = get_all_files.call(storage)
+  old_directory = storage.working_directory
   storage.save_changes!(files_to_commit: files_to_commit)
+
+
+  # need to check, because saving changes with delete is another function (update repo if needed)
+  files_diff = old_files - files_to_commit
+
+  if files_diff.length > 0
+    storage = storage_conf.call
+    encryption = encryption_conf.call(storage)
+
+    files_to_delete = files_diff.map do |file| 
+      old_file = file
+      old_file.slice! old_directory
+      new_file = File.join(storage.working_directory, old_file)
+      File.delete(new_file) if File.exist?(new_file)
+      file = new_file
+    end
+
+    encryption.encrypt_files
+    storage.save_changes!(files_to_delete: files_to_delete)
+  end
+
```

Comment:
call save_changes in `else` branch. it will delete temporary directory that was opened by finder